css/properties/display.json

```diff
@@ -1147,24 +1159,60 @@
               "ie": {
                 "version_added": false
               },
-              "opera": {
-                "version_added": "52"
-              },
-              "opera_android": {
-                "version_added": "47"
-              },
+              "opera": [
+                {
+                  "version_added": "52"
+                },
+                {
+                  "version_added": "45",
+                  "version_removed": "52",
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "Enable experimental Web Platform features"
+                    }
+                  ]
+                }
+              ],
+              "opera_android": [
+                {
+                  "version_added": "47"
+                },
+                {
+                  "version_added": "43",
+                  "version_removed": "47",
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "Enable experimental Web Platform features"
+                    }
+                  ]
+                }
+              ],
               "safari": {
                 "version_added": "11.1"
               },
               "safari_ios": {
                 "version_added": "11.1"
               },
               "samsunginternet_android": {
-                "version_added": false
+                "version_added": true
               },
-              "webview_android": {
-                "version_added": "65"
-              }
+              "webview_android": [
```

Comment:
webview_android has no flags. This would probably corrected later with the issue we have on file but maybe we want to avoid adding another instance of that here.

Reply:
This has now been fixed!